CONTRIBUTING

```diff
@@ -6,8 +6,7 @@ patches directly to the project.  In our collaborative open-source environment,
 standards and methods for submitting changes help reduce the chaos that can result
 from an active development community.
 
-This document briefly summarizes the full `Contribution
-Guidelines <http://projectacrn.github.io/latest/developer-guides/contribute_guidelines.html>`_
+This document briefly summarizes the full :ref:`contribute_acrn`
```

Comment:
I couldn't figure out a way to do this, it kept complaining about a duplication and removing the extension fixed it. Let me look into this a little further, I agree it would be better to be able to keep the extension.